packages/text-input-react/src/TextInput.tsx

```diff
@@ -22,6 +22,11 @@ export interface Props extends BaseProps {
     density?: Density;
     inline?: boolean;
     action?: Action;
+    /**
+     * Benevnelse for feltet. Unngå å bruke både benevnelse og handling samtidig
+     * @example "kr"
+     * */
+    unit?: ReactNode;
```

Comment:
Ser vi for oss et behov for noe annet enn `string` her?

Reply:
I eksemplet bruker jeg kvadratmeter, så jeg måtte kunne skrive `m<sup>2</sup>`. Siden jeg fant et case så fort tenkte jeg det kom til å bli nødvendig, hehe.